camille/util/resample.py

```diff
@@ -105,7 +105,7 @@ def resample(series, onto=None, interp='linear'):
         idx = onto
 
     ts = (
-        pd.concat([series, pd.Series(index=idx)])
+        pd.concat([series, pd.Series(index=idx, dtype=series.dtype)])
```

Comment:
Single commit, why is this done?

Reply:
Deprecation warning. Pandas series will require dtype in next version. Otherwise the type will be set as 'object', which none of our series are (Remember, they are 'float' on this level of timeseries)